code/game/gamemodes/objective.dm

```diff
@@ -689,6 +689,31 @@ GLOBAL_LIST_EMPTY(possible_items_special)
 		target_amount = count
 	update_explanation_text()
 
+/datum/objective/bankrupt
+	name = "bankrupt"
+	target_amount = 1000
+	var/target_department = ACCOUNT_CAR
+
+/datum/objective/bankrupt/update_explanation_text()
+	..()
+	explanation_text = "Bankrupt the station by ensuring the [SSeconomy.department_accounts[target_department]] has less than [target_amount] credits at the end of the shift."
+
+/datum/objective/bankrupt/check_completion()
+	var/datum/bank_account/D = SSeconomy.get_dep_account(target_department)
+	if(isnull(D) || D.account_balance <= target_amount)
+		return TRUE
+
+	return FALSE
+
+/datum/objective/bankrupt/admin_edit(mob/admin)
+	var/dep_choice = input(admin, "Which target department?", "Department Selection") as null|anything in SSeconomy.department_accounts
```

Comment:
I need to double-check this when it isn't 2am because there's a good chance it won't work.

Reply:
(I'm actually fairly certain it doesn't)

Reply:
Actually it probably does.

Reply:
(This is why you don't code at 2-3am).